addons/medical_damage/functions/fnc_determineIfFatal.sqf

```diff
@@ -49,7 +46,7 @@ if (EGVAR(medical,fatalDamageSource) in [1, 2]) then {
     _bodyPartDamage params ["_headDamage", "_bodyDamage"];
 
     private _vitalDamage = ((_headDamage - _headThreshhold) max 0) + ((_bodyDamage - _bodyThreshhold) max 0);
-    private _chanceFatal = 1 - exp -((_vitalDamage/WEIBULL_L)^WEIBULL_K);
+    private _chanceFatal = 1 - exp -((_vitalDamage/FATAL_SUM_DAMAGE_WEIBULL_L)^FATAL_SUM_DAMAGE_WEIBULL_K);
     TRACE_3("",_bodyPartDamage,_vitalDamage,_chanceFatal);
 
     if (_chanceFatal > random 1) exitWith {
```

Comment:
```sqf
    if (_chanceFatal > random 1 || _vitalDamage == 1) exitWith {
```
Since `_chanceFatal` will never reach 1 (and could be << 1 for x=1, if k and λ allow it), this would guarantee death.

Reply:
_vitalDamage isn't capped at 1
```
[ACE] (medical_damage) TRACE: 25999 : _bodyPartDamage=[3.68512,0,0,0,0,0], _vitalDamage=2.43512, _chanceFatal=1 z\ace\addons\medical_damage\functions\fnc_determineIfFatal.sqf:37